### ISSUE TITLE
Requesting child transaction records returns parent transaction record

### ISSUE THREAD

**Issue by SimiHunjan** [hashgraph/hedera-sdk-java#826]
### Description

Requesting a child transaction record incorrectly returns the parent transaction record.

### Steps to reproduce

Request a child transaction record

### Additional context

The list of child transaction records from the parent transfer transaction `[TransactionRecord{receipt=TransactionReceipt{status=SUCCESS, exchangeRate=ExchangeRate{hbars=0, cents=0, expirationTime=1970-01-01T00:00:00Z}, accountId=0.0.2914, fileId=null, contractId=null, topicId=null, tokenId=null, topicSequenceNumber=null, topicRunningHash=null, totalSupply=0, scheduleId=null, scheduledTransactionId=null, serials=[], duplicates=[], children=[]}, transactionHash=5adddd1766da8146c7f6e3a3abb702500af20a3e9adec75da51c7599a270c48480d6d3b8c481c9fcf5ec0c59f0149373, consensusTimestamp=2022-01-05T06:11:26.697466761Z, transactionId=0.0.2116@1641363076.500969845/1, transactionMemo=auto-created account, transactionFee=0 tℏ, contractFunctionResult=null, transfers=[], tokenTransfers={}, tokenNftTransfers={}, scheduleRef=null, assessedCustomFees=[], automaticTokenAssociations=[], aliasKey=302a300506032b65700321000ca2cb1757f7c63467e59954aecd09710a8f2c76e07aaa8287fe3921f3339d39, children=[], duplicates=[], parentConsensusTimestamp=null}]`

### Hedera network

previewnet

### Version

v2.6.0-beta.2

**Comment by Sean-Tedrow-LB** [comment]
Setting `getChildren(true)` on a `TransactionRecordQuery` _should_ cause the query to return the parent `TransactionRecord`, but child transactions should be included in that `TransactionRecord`'s `children` field.

Am I to understand that the `TransactionRecord.children` field is being populated with a copy of the parent transaction itself?  I don't see anything in the SDK source code that could result in that behavior.

**Comment by SimiHunjan** [comment]
Sorry I meant to close this. I chatted with Daniel on slack.

I thought that the parent consensus timestamp would be filled for the child account create transaction.

I learned that the parent consensus timestamp is **not** populated for the child account create transaction because it occurs before the parent transfer transaction. So it isn't a normal parent child relationship.

In other parent children relationships you would expect to see that field being filled for all children that occur after the parent is executed.

So this is correct as is.
